app/controllers/provider/admin/backend_apis/mapping_rules_controller.rb

```diff
@@ -1,14 +1,45 @@
 # frozen_string_literal: true
 
 class Provider::Admin::BackendApis::MappingRulesController < Provider::Admin::BackendApis::BaseController
+  include ThreeScale::Search::Helpers
+  include ProxyRuleSharedController
+
+  before_action :find_proxy_rule, only: %i[edit update destroy]
```

Comment:
This could be injected in `included` by `ProxyRuleSharedController`, no?

Reply:
makes total sense.. will try if it will works on included